KeyboardSwitch.Settings/Views/MainWindow.axaml.cs

```diff
@@ -38,15 +37,8 @@ public MainWindow()
                     .InvokeCommand(this.ViewModel.OpenAboutTab)
                     .DisposeWith(disposables);
             });
-
-            this.InitializeComponent();
-#if DEBUG
-            this.AttachDevTools();
-#endif
```

Comment:
Worth noting that Avalonia.NameGenerator is clever enough to generate `this.AttachDevTools()` on its own e.g. https://github.com/AvaloniaUI/Avalonia.NameGenerator/blob/main/src/Avalonia.NameGenerator.Tests/InitializeComponent/GeneratedInitializeComponent/AttachedPropsWithDevTools.txt